src/overlays/actors/ovl_En_Col_Man/z_en_col_man.h

```diff
@@ -8,10 +8,22 @@ struct EnColMan;
 typedef void (*EnColManActionFunc)(struct EnColMan*, GlobalContext*);
 
 typedef struct EnColMan {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ EnColManActionFunc actionFunc;
-    /* 0x0148 */ char unk_144[0x58];
-} EnColMan; // size = 0x1A0
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ EnColManActionFunc actionFunc;
+    /* 0x148 */ s16 unk148; // never used, only set
+    /* 0x14A */ char unk14A[0x2];
+    /* 0x14C */ s32 hasSetRandomValues;
```

Comment:
```suggestion
    /* 0x148 */ s16 unk148; // never used, only set
    /* 0x14C */ s32 hasSetRandomValues;
```
Could this just be padding?